Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: clean deps fmt vet test docker
+
+EXECUTABLE ?= drone-pypi
+IMAGE ?= plugins/$(EXECUTABLE)
+CI_BUILD_NUMBER ?= 0
+
+LDFLAGS = -X "main.buildDate=$(shell date -u '+%Y-%m-%d %H:%M:%S %Z')"
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+
+clean:
```

Comment:
This is already integrated into most plugins like it is now. The makefile have been discussed with brad. For a default task we can create a followup pull request series.